reference/configuration/framework.rst

```diff
@@ -1641,11 +1639,13 @@ To see a list of all available storages, run:
 handler_id
 ..........
 
-**type**: ``string`` **default**: ``session.handler.native_file``
+**type**: ``string`` or ``null`` **default**: ``null``
 
 The service id used for session storage. The default value ``'session.handler.native_file'``
```

Comment:
By default, the session handler configured by php.ini is used, unless option save_path is used, in which case the default is to store sessions using the native file session handler.